packages/nova-react/src/eventing/nova-eventing-provider.tsx

```diff
@@ -142,7 +151,7 @@ const generateEventing =
     generateEvent: (eventWrapper: GeneratedEventWrapper) => {
       const mappedEvent = {
         event: eventWrapper.event,
-        source: {
+        source: eventWrapper.source ?? {
```

Comment:
This change is a little weird, because we already support the timeStampOverride here, which is now completely ignored if the optional source is passed in.

Reply:
This kind of comes down to what we want the behaviour to be? :D 

In this case it seems we just want to "forward" the event to a handler further up in the tree. In that case should the overrides apply as initially the event was likely created using `bubble`?

But perhaps we need another way to just "bubble" those events further? It's almost like just doing this?
```ts 
forward: (eventWrapper: EventWrapper) => {
  return eventingRef.current.bubble(eventWrapper);
},
```

Reply:
> This change is a little weird, because we already support the timeStampOverride here, which is now completely ignored if the optional source is passed in.

We can also turn it around to check for timeStampOverride first, and then source. otherwise fallback to default which is `{inputType: InputType.programmatic, timeStamp:  Date.now()}`

Reply:
> This kind of comes down to what we want the behaviour to be? :D
> 
> In this case it seems we just want to "forward" the event to a handler further up in the tree. In that case should the overrides apply as initially the event was likely created using `bubble`?
> 
> But perhaps we need another way to just "bubble" those events further? It's almost like just doing this?
> 
> ```ts
> forward: (eventWrapper: EventWrapper) => {
>   return eventingRef.current.bubble(eventWrapper);
> },
> ```

this is kind of what we wanted to achieve.